lib/pure/complex.nim

```diff
@@ -17,96 +19,124 @@ import math
 
 type
   Complex*[T: SomeFloat] = object
-    re*, im*: T
     ## A complex number, consisting of a real and an imaginary part.
+    re*, im*: T
   Complex64* = Complex[float64]
-    ## Alias for a pair of 64-bit floats.
+    ## Alias for a complex number using 64-bit floats.
   Complex32* = Complex[float32]
-    ## Alias for a pair of 32-bit floats.
+    ## Alias for a complex number using 32-bit floats.
 
 func complex*[T: SomeFloat](re: T; im: T = 0.0): Complex[T] =
+  ## Returns a `Complex[T]` with real part `re` and imaginary part `im`
+  ## (default: 0).
```

Comment:
`## (default: 0).` is redundant; docs should only complement what's not implied in the declaration otherwise it's just noise

![image](https://user-images.githubusercontent.com/2194784/103600281-e3a4a080-4ebb-11eb-9ea7-d8d8cee68f85.png)